code/modules/clothing/suits/labcoat.dm

```diff
@@ -63,16 +63,34 @@
 	desc = "A suit that protects against minor chemical spills. Has an orange stripe on the shoulder."
 	icon_state = "labcoat_chem"
 
+/obj/item/clothing/suit/toggle/labcoat/chemist/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/chemistry,
+	)
+
 /obj/item/clothing/suit/toggle/labcoat/virologist
 	name = "virologist labcoat"
 	desc = "A suit that protects against minor chemical spills. Has a green stripe on the shoulder."
 	icon_state = "labcoat_vir"
 
+/obj/item/clothing/suit/toggle/labcoat/virologist/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/bio,
+	)
```

Comment:
```suggestion
	allowed += /obj/item/storage/bag/bio
```

Reply:
Will change and push them once I get connection back